service/ocean/providers/aws/cluster.go

```diff
@@ -162,21 +162,22 @@ type Filters struct {
 }
 
 type LaunchSpecification struct {
-	AssociatePublicIPAddress *bool                    `json:"associatePublicIpAddress,omitempty"`
-	AssociateIPv6Address     *bool                    `json:"associateIpv6Address,omitempty"`
-	SecurityGroupIDs         []string                 `json:"securityGroupIds,omitempty"`
-	ImageID                  *string                  `json:"imageId,omitempty"`
-	KeyPair                  *string                  `json:"keyPair,omitempty"`
-	UserData                 *string                  `json:"userData,omitempty"`
-	IAMInstanceProfile       *IAMInstanceProfile      `json:"iamInstanceProfile,omitempty"`
-	Tags                     []*Tag                   `json:"tags,omitempty"`
-	LoadBalancers            []*LoadBalancer          `json:"loadBalancers,omitempty"`
-	RootVolumeSize           *int                     `json:"rootVolumeSize,omitempty"`
-	Monitoring               *bool                    `json:"monitoring,omitempty"`
-	EBSOptimized             *bool                    `json:"ebsOptimized,omitempty"`
-	UseAsTemplateOnly        *bool                    `json:"useAsTemplateOnly,omitempty"`
-	InstanceMetadataOptions  *InstanceMetadataOptions `json:"instanceMetadataOptions,omitempty"`
-	LaunchSpecScheduling     *LaunchSpecScheduling    `json:"scheduling,omitempty"`
+	AssociatePublicIPAddress *bool                        `json:"associatePublicIpAddress,omitempty"`
+	AssociateIPv6Address     *bool                        `json:"associateIpv6Address,omitempty"`
+	SecurityGroupIDs         []string                     `json:"securityGroupIds,omitempty"`
+	ImageID                  *string                      `json:"imageId,omitempty"`
+	KeyPair                  *string                      `json:"keyPair,omitempty"`
+	UserData                 *string                      `json:"userData,omitempty"`
+	IAMInstanceProfile       *IAMInstanceProfile          `json:"iamInstanceProfile,omitempty"`
+	Tags                     []*Tag                       `json:"tags,omitempty"`
+	LoadBalancers            []*LoadBalancer              `json:"loadBalancers,omitempty"`
+	RootVolumeSize           *int                         `json:"rootVolumeSize,omitempty"`
+	Monitoring               *bool                        `json:"monitoring,omitempty"`
+	EBSOptimized             *bool                        `json:"ebsOptimized,omitempty"`
+	UseAsTemplateOnly        *bool                        `json:"useAsTemplateOnly,omitempty"`
+	InstanceMetadataOptions  *InstanceMetadataOptions     `json:"instanceMetadataOptions,omitempty"`
+	BlockDeviceMappings      []*ClusterBlockDeviceMapping `json:"blockDeviceMappings,omitempty"`
```

Comment:
change to ClusterBlockDeviceMappings

Reply:
Update name to ClusterBlockDeviceMappings